static-pages/about.md

```diff
@@ -1,3 +1,7 @@
 title: About crowd.loc.gov
 
+
 The Library of Congress launched crowd.loc.gov in the autumn of 2018. The application asks people to transcribe and tag digitized images of manuscripts and typed materials from the Library’s collections. Everyone is welcome to take part! You don't even need to create an account, but if you do you'll have access to additional features such as tagging and reviewing other people's transcriptions. All transcriptions are made by volunteers and reviewed by volunteers before they are returned to [loc.gov](https://loc.gov/), the Library's catalog. These transcriptions will improve search, readability, and access to handwritten and typed documents for those who are not fully sighted or cannot read the handwriting of the original documents.
+
+This program is generously supported by the National Digital Trust. This application is the result of collaboration between numerous divisions at the Library, including library services, curatorial and content specialists, digital preservation and web development specialists, digital innovation and strategy specialists.
```

Comment:
I think we should either list professions or divisions but not mix both with “library services”, which looks like it should be capitalized

Reply:
Also, if we can avoid referring to ourselves as "the Library" I think that would be a good thing. People who work in other libraries don't much appreciate that term.

Reply:
@acdha @rstorey good feedback, thanks